tests/integration/models/reddit/test_wikipage.py

```diff
@@ -72,6 +84,16 @@ def test_revisions(self, _):
                 assert isinstance(revision['page'], WikiPage)
             assert count > 0
 
+    @mock.patch('time.sleep', return_value=None)
+    def test_revisions__author_deleted(self, _):
+        subreddit = self.reddit.subreddit(
+            pytest.placeholders.test_subreddit)
+
+        with self.recorder.use_cassette(
+                'TestWikiPage.test_revisions__author_deleted'):
+            revisions = subreddit.wiki['index'].revisions(limit=10)
+            assert len(list(revisions)) == 10
```

Comment:
Is this the best way to test it? I think all I want to do is 1) ensure we've actually fetched the revision listing and 2) ensure there was no TypeError.